content/cloud/labels.md

```diff
@@ -0,0 +1,116 @@
+
+```bash
+spin cloud deploy
+Uploading todo-v2 version 0.1.0-r88287985 to Fermyon Cloud...
+Deploying...
+App "todo-app" accesses a database labeled "data"
+Would you like to link an existing database or create a new database?: Use an existing database and link app to it
+Which database would you like to link to todo-v2 using the label "data": todo_data
+App "todo-app" accesses a database labeled "data"
+```
+
+Let's kick things up a notch and deploy a second Spin application that will also use the todo_data database.
+
+```toml
+<...>
+name = "api-app"
+[component]
+id = "todo"
+sqlite_databases = ["defeault"]
+<.../>
+```
+
+The second Spin application is using its own label, default. 
+
+```bash
+spin cloud deploy
+Uploading api-app version 0.1.0-r88287985 to Fermyon Cloud...
+Deploying...
+App "api-app" accesses a database labeled "default"
+Would you like to link an existing database or create a new database?: Use an existing database and link app to it
+Which database would you like to link to todo-v2 using the label "default": todo_data
+App "api-app" accesses a database labeled "data"
+```
+
+Now we have two seperate Spin applications using their own labels can link to the same logical resource. 
+
+As illustrated in this example, links and labels are application scoped. Whenever that Spin application needs to access that resource, it will use the label as reference. This also means a NoOps SQL DB can be linked to multiple Spin applications which all use their own respective labels.
+
+The Spin application does not need to know the true name of the resource it's interacting with. This also means a label is well understood by all the components within it's respective application, but it doesn't hold any significance to other applications. 
+
+> At this time, only NoOps SQL DB supports labels. However, in the future other Fermyon Cloud resources such as Key Value Store will support links and labels as well. 
+
+## Benefits of Links and Labels
+
+Benefits of the Fermyon Clould links and labels include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
+* **Flexible Lifecycle Management**: Manage Fermyon Cloud resources independently of your Spin application.
+* **Seamless Cloud Integration**: Cloud experience smoothly integrates with local Spin development, no runtime configuration changes required. 
+* **Dynamic App Resource Configuration**: Change the data resource your application points to without rebuilding it.
+
+## Working With Links and Labels
+
+Whenever you add a string to your NoOps SQL database entry in the component manifest, you're creating a label. When you deploy your Spin application to Fermyon Cloud, you'll be prompted to link the label to a specific NoOps SQL database instance. Later on, if you'd like to update the link you can do so via the `spin cloud link` command. 
```

Comment:
This points out that we may want to allow updating links in `spin cloud link`. i realized we dont currently and you have to unlink first
```suggestion
Whenever you add a string to your NoOps SQL database entry in the component manifest, you're creating a label. When you deploy your Spin application to Fermyon Cloud, you'll be prompted to link the label to a specific NoOps SQL database instance. Later on, if you'd like to update the link you can do so via the `spin cloud unlink` and then `spin cloud link` commands. 
```